buildSrc/src/main/kotlin/Testing.kt

```diff
@@ -87,6 +87,7 @@ class NessieTestingPlugin : Plugin<Project> {
       tasks.withType<Test>().configureEach {
         val testJvmArgs: String? by project
         val testHeapSize: String? by project
+        jvmArgs("-XX:+HeapDumpOnOutOfMemoryError")
```

Comment:
Will the dump be saved in CI runs? Is the expected benefit for local runs only?

Reply:
It is (was) for local use at the moment.